tests/BucketerTest.php

```diff
@@ -28,15 +28,12 @@
 
 class BucketerTest extends \PHPUnit_Framework_TestCase
 {
-    private $testBucketingIdControl;
     private $testUserId;
     private $config;
     private $loggerMock;
 
     public function setUp()
     {
-        $this->testBucketingIdControl = 'testBucketingIdControl!';  // generates bucketing number 3741
-        $this->testBucketingIdVariation = '123456789'; // generates bucketing number 4567
```

Comment:
I don't get how these Ids help in testing. The variables, at most times are passed in tests where we already mock bucket values by TestBucketer so these are ignored. At other times, where these ids do have an impact, we can not really predict the bucketing number because these values are concatenated with group id or experiment id (in a given test) before being passed to murmur hash method.